docs/migration/v8-to-v9.md

```diff
@@ -126,65 +128,66 @@ Sentry.init({
 
-### `@sentry/browser`
-
-- The `captureUserFeedback` method has been removed. Use `captureFeedback` instead and update the `comments` field to `message`.
-
 ### `@sentry/nestjs`
 
-- Removed `WithSentry` decorator. Use `SentryExceptionCaptured` instead.
+- Removed `WithSentry` decorator. Use the `SentryExceptionCaptured` decorator instead.
 - Removed `SentryService`.
-  If you are using `@sentry/nestjs` you can safely remove any references to the `SentryService`.
-  If you are using another package migrate to `@sentry/nestjs` and remove the `SentryService` afterwards.
+  - If you are using `@sentry/nestjs` you can safely remove any references to the `SentryService`.
+  - If you are using another package migrate to `@sentry/nestjs` and remove the `SentryService` afterward.
 - Removed `SentryTracingInterceptor`.
-  If you are using `@sentry/nestjs` you can safely remove any references to the `SentryTracingInterceptor`.
-  If you are using another package migrate to `@sentry/nestjs` and remove the `SentryTracingInterceptor` afterwards.
+  - If you are using `@sentry/nestjs` you can safely remove any references to the `SentryTracingInterceptor`.
+  - If you are using another package migrate to `@sentry/nestjs` and remove the `SentryTracingInterceptor` afterward.
 - Removed `SentryGlobalGenericFilter`.
-  Use the `SentryGlobalFilter` instead.
-  The `SentryGlobalFilter` is a drop-in replacement.
+  - Use the `SentryGlobalFilter` instead.
+  - The `SentryGlobalFilter` is a drop-in replacement.
 - Removed `SentryGlobalGraphQLFilter`.
-  Use the `SentryGlobalFilter` instead.
-  The `SentryGlobalFilter` is a drop-in replacement.
+  - Use the `SentryGlobalFilter` instead.
+  - The `SentryGlobalFilter` is a drop-in replacement.
+
+### `@sentry/react`
+
+- The `wrapUseRoutes` method has been removed. Use the `wrapUseRoutesV6` or `wrapUseRoutesV7` methods instead depending on what version of react router you are using.
+- The `wrapCreateBrowserRouter` method has been removed. Use the `wrapCreateBrowserRouterV6` or `wrapCreateBrowserRouterV7` methods depending on what version of react router you are using.
 
 ## `@sentry/vue`
 
 - The options `tracingOptions`, `trackComponents`, `timeout`, `hooks` have been removed everywhere except in the `tracingOptions` option of `vueIntegration()`.
-  These options should now be set as follows:
-
-  ```ts
-  import * as Sentry from '@sentry/vue';
-
-  Sentry.init({
-    integrations: [
-      Sentry.vueIntegration({
-        tracingOptions: {
-          trackComponents: true,
-          timeout: 1000,
-          hooks: ['mount', 'update', 'unmount'],
-        },
-      }),
-    ],
-  });
-  ```
+
+These options should now be set as follows:
+
+```js
+import * as Sentry from '@sentry/vue';
+
+Sentry.init({
+  integrations: [
+    Sentry.vueIntegration({
+      tracingOptions: {
+        trackComponents: true,
+        timeout: 1000,
+        hooks: ['mount', 'update', 'unmount'],
+      },
+    }),
+  ],
+});
+```
```

Comment:
l: should this not stay indented as it belongs to the bullet point above?

Reply:
yup! good catch